tests/RA_UnitTestHelpers.h

```diff
@@ -14,9 +14,15 @@ namespace CppUnitTestFramework {
 
 // converters for asserting enum values
 
-template<> static std::wstring ToString<ComparisonVariableSize>(const ComparisonVariableSize& t)
+template<> static std::wstring ToString<MemSize>(const MemSize& t)
 {
-    return ra::Widen(COMPARISONVARIABLESIZE_STR[(int)t]);
+#if _MBCS
+    return ra::Widen(MEMSIZE_STR.at(ra::etoi(t)));
+#elif _UNICODE
+    return MEMSIZE_STR.at(ra::etoi(t));
+#else
+#error Unsupported character set detected
+#endif /* _MBCS */
```

Comment:
don't the `ra::Widen` overloads handle these `#if`s?

Reply:
You are totally right. Have to look at again but we probably don't even need the widen.